packages/components/src/components/post-tab-panel/post-tab-panel.tsx

```diff
@@ -0,0 +1,28 @@
+import { Component, Element, h, Prop } from '@stencil/core';
+
+@Component({
+  tag: 'post-tab-panel',
+  styleUrl: 'post-tab-panel.scss',
+  shadow: true,
+})
+export class PostTabPanel {
+  @Element() host: HTMLPostTabPanelElement;
+
+  /**
+   * The name of the panel, used to associate it with a tab header.
+   */
+  @Prop() readonly name: string;
+
+  render() {
+    return (
+      <div
+        aria-labelledby={`${this.name}--tab`}
+        class="tab-pane fade active show"
+        id={`${this.name}--panel`}
```

Comment:
This id is not secure enough to be sure that there will never be duplicates.

@gfellerph @oliverschuerch @imagoiq 
Should we go for the id/for solution where devs will be responsible for providing unique ids or should we stick with the name/panel approach and create uniq ids ourselves?

Reply:
According to the resolution, keep the custom prop names for connecting header and panel and generate uinique ids for internal usage with `crypto.randomUUID` (but this will only work on https and localhost, not on http pages) or use a custom function for generating more or less random strings (always start with a letter in order to be compliant with id requirements).

Reply:
Ok, thanks for the clarification